app/helpers/application_helper.rb

```diff
@@ -244,4 +244,12 @@ def current_users_resetting_password_email
     # by_email_or_username
     !current_user.is_anonymous? && EmailAddress.verified.where(user: current_user).first.try(:value)
   end
+
+  def average_time_for_users_to_become_activated
+    query = "SELECT AVG(EXTRACT('EPOCH' FROM AGE(created_at, activated_at))) FROM users WHERE activated_at IS NOT NULL"
```

Comment:
suggestion-queries like this should live in the model (so that it can be reused and unit tested there)